ui/app/mainui/AppMain.qml

```diff
@@ -2135,6 +2137,7 @@ Item {
 
         width: appMain.width
         height: appMain.height
+        onDroppedOnValidScreen: (drop) => Global.droppedOnValidScreen(drop)
```

Comment:
I think that `DropAreaPanel` could and should be moved into `StatusChatInpput`. Then there is no need to use `Global` as a proxy, status chat input is self-contained regarding dnd support and no bindin to `chatInputEnabled` is needed then. I would give a try, if I'm not missing anything, it should simplify the code.

Reply:
Fixed; much cleaner now

Reply:
So nice! Now that functionality is in just 1 place instead of ~10 places :D